packages/gatsby/src/services/initialize.ts

```diff
@@ -41,18 +41,17 @@ if (
   process.env.GATSBY_EXPERIMENTAL_FAST_DEV &&
   !isCI()
 ) {
-  process.env.GATSBY_EXPERIMENTAL_LAZY_IMAGES = `true`
-  process.env.GATSBY_EXPERIMENTAL_QUERY_ON_DEMAND = `true`
   process.env.GATSBY_EXPERIMENTAL_DEV_SSR = `true`
+  process.env.PRESERVE_FILE_DOWNLOAD_CACHE = `true`
+  process.env.PRESERVE_WEBPACK_CACHE = `true`
```

Comment:
This is just "random" thing I spotted when looking for qod related changes I needed - we added those to `FAST_DEV` config flag but not to `GATSBY_EXPERIMENTAL_FAST_DEV` env var route - I can drop those from here, but it would be weird for `FAST_DEV` to only map to `DEV_SSR` 🤷